content/en/data_jobs/databricks.md

```diff
@@ -222,6 +222,14 @@ In Datadog, view the [Data Jobs Monitoring][6] page to see a list of all your Da
 
 ## Advanced Configuration
 
+### Filter log collection on a clusters
+
+#### Exclude all log collection from an individual cluster
+Configure the following environment variable in the Advanced Configuration section of your cluster in the Databricks UI or as [Spark env vars][2] in the Databricks API.
```

Comment:
```suggestion
Configure the following environment variable in the Advanced Configuration section of your cluster in the Databricks UI or as a [Spark environment variable][2] in the Databricks API.
```
Also, the link target `[2]` is defined above in a tab, so it's not active here. For the link to work, you need to add a line at the bottom of the file:
`[2]: https://docs.databricks.com/api/workspace/clusters/edit#spark_env_vars`